lighthouse-plugin-ad-speed-insights/gatherers/iframe-elements.js

```diff
@@ -19,6 +19,43 @@ const pageFunctions = require('lighthouse/lighthouse-core/lib/page-functions.js'
 
 /* eslint-env browser, node */
 
+/**
+ * TODO(jburger): Move to page-functions.js once integrated with Lighthouse.
+ * @param {?HTMLElement} element
+ * @return {Boolean}
+ */
+function isFixed(element) {
+  // @ts-ignore
+  const scrollableAncestor = ((element) => {
```

Comment:
Is it worth pulling this helper outside of this function? It doesn't look like it closes over anything. In fact it shadows the outer `element` which isn't great